src/pal/src/misc/tracepointprovider.cpp

```diff
@@ -87,26 +87,23 @@ PAL_InitializeTracing(void)
     {
         return;
     }
+    
+    SIZE_T tpLibNameLen = strlen(tpLibName);
 
-    // Make sure that the final path is shorter than MAX_PATH.
-    // +1 ensures that the string can be NULL-terminated.
-    if((lastTrailingSlashLen + tpLibNameLen + 1) > MAX_LONGPATH)
+    if( !tpProvPath.Reserve(tpLibNameLen + lastTrailingSlashLen) ||  
+    // Copy the path without the shared object name.
+        !tpProvPath.Append(info.dli_fname, lastTrailingSlashLen) ||
```

Comment:
This is being extra paranoid and not fail here ever, this codepath loads the Lttng logging modules and is something which we can go on even if it fails